README: Replace instructions with link to docs (#110)
circleci/enterprise-setup: README.md

# CircleCI Server Setup

This package allows you to easily orchestrate your CircleCI Server cluster in AWS using Terraform.

**Note: This is only meant to be used for the initial setup of CircleCI Server and is not meant to be used for the ongoing maintenance of the CircleCI Server.**

**Note: Master is the only supported branch. All other branches of this repo should not be considered stable, and is to be used at your own risk.**

> ## Documentation

Reviewer: Might be worth pointing people to the Prerequisites and Planning doc as well the install guide? https://circleci.com/docs/2.0/aws-prereq/#section=server-administration. This details needing terraform, and all the other bits of info they are going to need to have to hand.

Reviewer 2: Done

# CircleCI Server Setup

> This package allows you to easily orchestrate your CircleCI Server cluster in AWS using Terraform.

Reviewer: ```suggestion
This package allows you to orchestrate your CircleCI Server cluster in AWS using Terraform.
```

**Note: This is only meant to be used for the initial setup of CircleCI Server and is not meant to be used for the ongoing maintenance of the CircleCI Server.**

**Note: Master is the only supported branch. All other branches of this repo should not be considered stable, and is to be used at your own risk.**

## Prerequisites and Requirements

> Please refer to our prerequisites documentation here: https://circleci.com/docs/2.0/aws-prereq

Reviewer: ```suggestion
Please refer to our prerequisites documentation here: https://circleci.com/docs/2.0/aws-prereq.
```

## Installation

> You can find instructions here: https://circleci.com/docs/2.0/aws/

Reviewer: ```suggestion
You can find instructions here: https://circleci.com/docs/2.0/aws/.
```

  | services_user_data_enabled | Enable/disable automated installation on Services Box | true |
  | force_destroy_s3_bucket | Add/Remove ability to forcefully destroy S3 bucket | false |
  | services_disable_api_termination | Protect the services instance from API termination | true |

### Teardown

> You can find teardown instructions at https://circleci.com/docs/2.0/aws-teardown/

Reviewer: ```suggestion
You can find teardown instructions at https://circleci.com/docs/2.0/aws-teardown/.
```

Reviewer 2: Done

## Upgrade

> If you want to upgrade from a previous installation, please follow these [upgrade instructions](https://circleci.com/docs/2.0/updating-server/#section=server-administration)

Reviewer: ```suggestion
If you want to upgrade an existing installation from a previous version, follow these [upgrade instructions](https://circleci.com/docs/2.0/updating-server/#section=server-administration).
```
